### PR TITLE
Updating response sections for phones /challenge endpoint

### PR DESCRIPTION
**Pull request by justinabrokwah-okta**
## Description:
- Updating the response body and providing samples for the `/idp/myaccount/phones/{id}/challenge` endpoint
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-485374](https://oktainc.atlassian.net/browse/OKTA-485374)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 1]
```diff
@@ -1152,7 +1152,11 @@ This request requires the `method` property as its request body. An optional boo
 
 #### Response body
 
-Returns an empty response with an HTTP 200 status code.
+Returns a JSON object containing a link to the verify endpoint an HTTP 200 status code.
```
[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1155]: Sorry for the confustion @brentschaus-okta ; what I want to say is that a link to verify the challenged phone is provided in the JSON object, how does this sound?
`Returns a JSON object containing a link to verify the challenged phone, with an HTTP 200 status code.`

[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md]: @justinabrokwah-okta i think that's great. thank you!

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

**Comment by justinabrokwah-okta** [comment]
> LGTM!

Thanks for the review @brentschaus-okta ! Should I merge it into the hero branch or will you do that?

**Pull request merged**